src/main/resources/db/migration/V4__add_fks_role_table.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE JobRoles
```

Comment:
why are we making a new migration to alter the jobroles table when it could just be added to the create migration

Reply:
Because of the order of the migrations, if foreign keys are added on the job roles table before the other tables exist it will error